extensions/sql-migration/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+	"extends": "../shared.tsconfig.json",
+	"compileOnSave": true,
+	"compilerOptions": {
+		"module": "commonjs",
+		"outDir": "./out",
+		"lib": [
+			"es6", "es2015.promise"
+		],
+		"emitDecoratorMetadata": true,
+		"experimentalDecorators": true,
+		"moduleResolution": "node",
+		"declaration": false,
+		"strict": false,
```

Comment:
Not enabling strict?

Reply:
Good idea, let me see if I can add some extra rules for this project too.